tests/test_plugin.py

```diff
@@ -604,9 +629,9 @@ def test_openchannel_hook_error_handling(node_factory, bitcoind):
     l1.fundwallet(10**6)
 
     # next fundchannel should fail fatal() for l2
-    with pytest.raises(RpcError, match=r'Owning subdaemon openingd died'):
+    with pytest.raises(RpcError, match=r'Owning subdaemon (openingd|dualopend) died'):
         l1.rpc.fundchannel(l2.info['id'], 100004)
-    assert l2.daemon.is_in_log("BROKEN.*Plugin rejected openchannel but also set close_to")
+    assert l2.daemon.is_in_log("BROKEN.*Plugin rejected openchannel2? but also set close_to")
```

Comment:
Just to make it clear what the `?` applies to:

```suggestion
    assert l2.daemon.is_in_log("BROKEN.*Plugin rejected openchannel[2]? but also set close_to")
```